### PR TITLE
Runtimed kernel run

### PR DESCRIPTION
**Pull request by cvaske**
Still to do:
- [x] signal handler for runtimed to clean up child processes
- [x] redirecty stdout/stderr to log files in runtime directory

### CODE REVIEW

**Comment by cvaske** [comment]
Holding off on stdout/stderr logging until review of the signal handling.

One potential issue with the SIGINT signal handler for killing child kernel processes: when SIGINT is sent to runtimed, `.start_kill()` will still be called on child processes that have already been terminated. I can't yet find a problem with doing this, and it appears to cause no problems, but I also haven't found any documentation that it's explicitly OK.

File: .ipynb_checkpoints/Untitled-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+ "cells": [],
+ "metadata": {},
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```
Comment (rgbkrk) [inline .ipynb_checkpoints/Untitled-checkpoint.ipynb:6]: Might need this directory in a gitignore, along with Untitled*.ipynb.

**Comment by cvaske** [comment]
@rgbkrk This should be complete, can you give me feedback?

File: runtimed/src/runtime_manager.rs [hunk 2]
```diff
@@ -75,41 +111,148 @@ impl RuntimeManager {
...
+        let k = runtimelib::jupyter::KernelspecDir::new(kernel_name).await?;
+        let ci = runtimelib::jupyter::client::ConnectionInfo::new("127.0.0.1", kernel_name).await?;
+        let connection_file_path = ci.generate_file_path();
+        let runtime = JupyterRuntime::new(ci, connection_file_path);
+        let mut command = k.command(&runtime.connection_file).await?;
+
+        // Insert the runtime into the RuntimeManager before writing the connection file
+        // because the watcher will try to insert the runtime into the hashmap, but will
+        // not have access to the ChildRuntime process handle
+        //self.insert(&runtime, None).await?;
```
Comment (rgbkrk) [inline runtimed/src/runtime_manager.rs:175]: This line can be deleted now

**Review by rgbkrk — APPROVED** [review]
There are some TODOs that can become issues now. Otherwise, looking great.

**Pull request merged**